lib/livebook.ex

```diff
@@ -114,6 +114,11 @@ defmodule Livebook do
       config :livebook, LivebookWeb.Endpoint, url: [path: base_url_path]
     end
 
+    if public_base_url_path_override =
+         Livebook.Config.public_base_url_path_override!("LIVEBOOK_PUBLIC_BASE_URL_PATH_OVERRIDE") do
```

Comment:
We can use `Livebook.Config.base_url_path!/1`, since both are of the same "type" and follow the same parsing logic.

Also, let's remove "override" from the naming :)

Reply:
Done :)